config/config.default.yaml

```diff
@@ -10,6 +10,14 @@ logging:
   level: INFO
   format: '%(levelname)s:%(name)s:%(message)s'
 
+private:
+  keys:
+    entsoe_api:
```

Comment:
It might be wise to implement it in a way to reference a file, like `.cdsapirc`, since otherwise, people will definitely push their private API keys.

Reply:
I would not fear that people put their private key in the config.default.yaml. In the config.yaml yes, but this is ignored anyway.  I can imagine it is good to have a private section, as this might be not the last case where we need a api key in the future.